dcc-portal-ui/app/scripts/advanced/js/advanced.js

```diff
@@ -17,6 +17,8 @@
 
 'use strict';
 
+import ensureString from '../../common/js/ensure-input';
```

Comment:
Yes it needs to be in braces. However, that function wasn't in used so I removed its instance.